### ISSUE TITLE
Error while Building using Microsoft AppCenter

### ISSUE THREAD

**Issue by XFAISAL** [maplibre/maplibre-gl-native-distribution#11]
Hello there im trying to build the app using Appcenter and every time is stuck after this line 

```
Build settings from command line:
    CODE_SIGNING_ALLOWED = NO

Resolve Package Graph

Fetching from https://github.com/maplibre/maplibre-gl-native-distribution

Cloning local copy of package ‘maplibre-gl-native-distribution’

Checking out 5.12.1 of package ‘maplibre-gl-native-distribution’

##[error]The operation was canceled.
```

**Comment by roblabs** [comment]
Check the version as the most recent release is different than what the log is reporting.  Although, previous versions should work with Swift Package Manager.  (or in MSFT AppCenter, in this case).

https://github.com/maplibre/maplibre-gl-native-distribution/blob/main/Package.swift#L13,L17

<img src="https://user-images.githubusercontent.com/118112/202246671-487aa7d8-b94d-431d-af41-c15e4bc05f5a.jpeg" width="25%">

**Comment by XFAISAL** [comment]
Updated and still same issue

`Fetching from https://github.com/maplibre/maplibre-gl-native-distribution

Cloning local copy of package ‘maplibre-gl-native-distribution’

Checking out 5.12.2 of package ‘maplibre-gl-native-distribution’

##[error]The operation was canceled.`

**Comment by harry-isaac** [comment]
@XFAISAL Did you ever figure this out? I'm experiencing the same issue

**Comment by XFAISAL** [comment]
no results until now @harry-isaac @roblabs

**Comment by XFAISAL** [comment]
This is MS Center Replay :


 
Hi there,﻿Thanks for reaching out to us!It seems that you are using Swift Package Manager to manage project dependencies in your IOS project. App Center doesn’t fully support SPM currently.As a result, the issue occurs.
https://github.com/microsoft/appcenter/issues/560
https://github.com/microsoft/appcenter/issues/2408
https://github.com/microsoft/appcenter/issues/2414

I am sorry for the inconvenience.Feel free to let us know if there is anything else we can help with.Thanks

**Comment by harry-isaac** [comment]
Ugh okay, thanks! Guess it's time to move to EAS...

**Comment by hexadecy** [comment]
I only have this error with App Center when building with GitHub.
It's ok when building with a self-hosted Gitlab. 🤷

**Comment by fbarbat** [comment]
We hit this issue as well. I attach a repro case.
I created an Xcode project from scratch and added a single Swift package: https://github.com/stasel/WebRTC
Then, I created an AppCenter build for it and it always hangs. 
When I was trying to debug it, I tried to compile the project from the command line and a dialog like this would show up:
![image](https://github.com/maplibre/maplibre-gl-native-distribution/assets/857775/3d6fbe69-bed4-4b41-a6f8-3ff386a8409b)
This is because this project has a Swift Package that uses a binaryTarget pointing to a GitHub URL and xcodebuild is trying to access the keychain. I could see in the debug logs that AppCenter creates an internal keychain and unlocks it, but I think the xcodebuild command is not using it. So AppCenter is probably hanging with the dialog I sent you before (but as it doesn't have a UI, we can't see it of course).
